### PR TITLE
docs: translating renderToStaticNodeStream

### PR DESCRIPTION
**Pull request by rimamei**
Closes https://github.com/reactjs/id.react.dev/issues/461

translation to Bahasa for renderToStaticNodeStream.

Reference: https://github.com/reactjs/id.react.dev/issues/330

### CODE REVIEW

File: src/content/reference/react-dom/server/renderToStaticNodeStream.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: renderToStaticNodeStream
 
 <Intro>
 
-`renderToStaticNodeStream` renders a non-interactive React tree to a [Node.js Readable Stream.](https://nodejs.org/api/stream.html#readable-streams)
+`renderToStaticNodeStream` merender sebuah pohon React yang tidak interaktif ke dalam sebuah [*Node.js Readable Stream*.](https://nodejs.org/api/stream.html#readable-streams)
```
Comment (r17x) [inline src/content/reference/react-dom/server/renderToStaticNodeStream.md:7]:
```suggestion
`renderToStaticNodeStream` me-*render* sebuah pohon (*tree*) React yang tidak interaktif ke dalam sebuah [*Node.js Readable Stream*.](https://nodejs.org/api/stream.html#readable-streams)
```

File: src/content/reference/react-dom/server/renderToStaticNodeStream.md [hunk 2]
```diff
@@ -29,52 +29,52 @@ const stream = renderToStaticNodeStream(<Page />);
...
+[Lihat lebih banyak contoh di bawah ini.](#usage)
 
-The stream will produce non-interactive HTML output of your React components.
+*Stream* akan menghasilkan keluaran HTML yang tidak interaktif dari komponen-komponen React kalian.
 
-#### Parameters {/*parameters*/}
+#### Parameter {/*parameters*/}
 
-* `reactNode`: A React node you want to render to HTML. For example, a JSX element like `<Page />`.
+* `reactNode`: Sebuah *node* React yang ingin kalian render ke dalam HTML. Misalnya, sebuah elemen JSX seperti `<Page />`.
```
Comment (r17x) [inline src/content/reference/react-dom/server/renderToStaticNodeStream.md:38]:
```suggestion
* `reactNode`: Sebuah *node* React yang ingin kalian *render* ke dalam HTML. Misalnya, sebuah elemen JSX seperti `<Page />`.
```

File: src/content/reference/react-dom/server/renderToStaticNodeStream.md [hunk 3]
```diff
@@ -29,52 +29,52 @@ const stream = renderToStaticNodeStream(<Page />);
...
 
 #### Returns {/*returns*/}
 
-A [Node.js Readable Stream](https://nodejs.org/api/stream.html#readable-streams) that outputs an HTML string. The resulting HTML can't be hydrated on the client.
+Sebuah [*Node.js Readable Stream*](https://nodejs.org/api/stream.html#readable-streams) yang menghasilkan sebuah *string* HTML. HTML yang dihasilkan tidak bisa dihidrasi di sisi klien.
```
Comment (r17x) [inline src/content/reference/react-dom/server/renderToStaticNodeStream.md:42]:
```suggestion
Sebuah [*Node.js Readable Stream*](https://nodejs.org/api/stream.html#readable-streams) yang menghasilkan sebuah *string* HTML. HTML yang dihasilkan tidak bisa dihidrasi (*hydrated*) di sisi klien.
```

File: src/content/reference/react-dom/server/renderToStaticNodeStream.md [hunk 4]
```diff
@@ -29,52 +29,52 @@ const stream = renderToStaticNodeStream(<Page />);
...
 
-#### Caveats {/*caveats*/}
+#### Batasan {/*caveats*/}
 
-* `renderToStaticNodeStream` output cannot be hydrated.
+* Keluaran `renderToStaticNodeStream` tidak dapat dihidrasi.
```
Comment (r17x) [inline src/content/reference/react-dom/server/renderToStaticNodeStream.md:46]:
```suggestion
* Keluaran `renderToStaticNodeStream` tidak dapat dihidrasi (*hydrated*).
```

File: src/content/reference/react-dom/server/renderToStaticNodeStream.md [hunk 5]
```diff
@@ -29,52 +29,52 @@ const stream = renderToStaticNodeStream(<Page />);
...
 
-* This method will wait for all [Suspense boundaries](/reference/react/Suspense) to complete before returning any output.
+* Metode ini akan menunggu semua [*Suspense boundaries*](/reference/react/Suspense) untuk diselesaikan sebelum mengembalikan keluaran apapun.
 
-* As of React 18, this method buffers all of its output, so it doesn't actually provide any streaming benefits.
+* Sejak React 18, metode ini menyangga semua keluarannya, sehingga tidak memberikan keuntungan *stream* apapun .
 
-* The returned stream is a byte stream encoded in utf-8. If you need a stream in another encoding, take a look at a project like [iconv-lite](https://www.npmjs.com/package/iconv-lite), which provides transform streams for transcoding text.
+* *Stream* yang dikembalikan merupakan sebuah *byte stream* yang dienkode dalam utf-8. Jika kalian membutuhkan sebuah *stream* yang dienkode lainnya, lihatlah ke sebuah proyek seperti [iconv-lite](https://www.npmjs.com/package/iconv-lite), yang memberikan transformasi aliran untuk *transcoding* teks.
```
Comment (r17x) [inline src/content/reference/react-dom/server/renderToStaticNodeStream.md:52]:
```suggestion
* *Stream* yang dikembalikan merupakan sebuah *byte stream* yang *encoded* dalam *utf-8*. Jika kalian membutuhkan sebuah *stream* lain yang di-*encode*, lihatlah ke sebuah proyek seperti [iconv-lite](https://www.npmjs.com/package/iconv-lite), yang memberikan transformasi aliran untuk *transcoding* teks.
```

File: src/content/reference/react-dom/server/renderToStaticNodeStream.md [hunk 6]
```diff
@@ -29,52 +29,52 @@ const stream = renderToStaticNodeStream(<Page />);
...
 
 ---
 
-## Usage {/*usage*/}
+## Penggunaan {/*usage*/}
 
-### Rendering a React tree as static HTML to a Node.js Readable Stream {/*rendering-a-react-tree-as-static-html-to-a-nodejs-readable-stream*/}
+### Merender sebuah pohon React sebagai HTML statis ke dalam sebuah *Node.js Readable Stream* {/*rendering-a-react-tree-as-static-html-to-a-nodejs-readable-stream*/}
```
Comment (r17x) [inline src/content/reference/react-dom/server/renderToStaticNodeStream.md:58]:
```suggestion
### Me-render sebuah pohon React sebagai HTML statis ke dalam sebuah Node.js Readable Stream {/*rendering-a-react-tree-as-static-html-to-a-nodejs-readable-stream*/}
```

File: src/content/reference/react-dom/server/renderToStaticNodeStream.md [hunk 7]
```diff
@@ -29,52 +29,52 @@ const stream = renderToStaticNodeStream(<Page />);
...
 });
 ```
 
-The stream will produce the initial non-interactive HTML output of your React components.
+*Stream* akan menghasilkan keluaran awal HTML yang tidak interaktif dari komponen-komponen React kalian.
 
 <Pitfall>
 
-This method renders **non-interactive HTML that cannot be hydrated.** This is useful if you want to use React as a simple static page generator, or if you're rendering completely static content like emails.
+Metode ini merender **HTML tidak interaktif yang tidak bisa dihidrasi.** Ini berguna jika kalian ingin menggunakan React sebagai sebuah generator halaman statis sederhana, atau jika kalian merender konten statis seperti email.
```
Comment (r17x) [inline src/content/reference/react-dom/server/renderToStaticNodeStream.md:76]:
```suggestion
*Method* ini me-*render* **HTML tidak interaktif yang tidak bisa dihidrasi (*hydrated*).** Ini berguna jika kalian ingin menggunakan React sebagai sebuah *generator* halaman statis sederhana, atau jika kalian me-*render* konten statis seperti email.
```

**Review by r17x — CHANGES_REQUESTED** [review]

**Comment by rimamei** [comment]
There's another change that I add. I change `kalian` to `anda` as other PR use anda and use `caveats` as it is based on this PR review https://github.com/reactjs/id.react.dev/pull/364

**Review by r17x — APPROVED** [review]

**Pull request merged**